src/index.js

```diff
@@ -760,18 +787,53 @@ import { isRemote } from 'pouchdb-utils';
                             .on('update', (update, aggregate) => {
                                 if (first && aggregate)
                                     aggregate = aggregate[0];
+
                                 vm.$data[key] = aggregateCache = aggregate;
+
+                                vm.$emit('pouchdb-livefeed-update', {
+                                    db: key,
+                                    name: db.name
+                                });
+    
                             })
                             .on('ready', () => {
                                 vm.$data[key] = aggregateCache;
-                            });
+
+                                vm.$emit('pouchdb-livefeed-ready', {
+                                    db: key,
+                                    name: db.name
+                                });
+                            })
+                            .on('cancelled', function() {
+                                vm.$emit('pouchdb-livefeed-cancel', {
+                                    db: key,
+                                    name: db.name,                                                                        
+                                    name: db.name
+                                });
+                              })
+                              .on('error', function(err) {
+                                vm.$emit('pouchdb-livefeed-error', {
+                                    db: key,
+                                    name: db.name,                                    
+                                    errorobject: err,
```

Comment:
maybe pass the err to error: instead of errorobject, since its the real error

Reply:
done